README.md

```diff
@@ -35,6 +35,7 @@ Below is a simple Javascript express app using TrueLayer's API client library. I
 ```javascript
 const {AuthAPIClient, DataAPIClient} = require("truelayer-client");
 const app = require("express")();
+const parse = require("body-parser")
```

Comment:
can we avoid depending on `body-parser` in the sample for simplicity? We can also demo it with GET since it is the new default